app/components/searchable_collection_component/searchable_collection_component.html.slim

```diff
@@ -2,7 +2,7 @@
   div class="#{border_class} #{scrollable_class}"
     - if searchable?
       .searchable-collection-component__search
-        input.govuk-input.icon.icon--left.icon--search.js-action data-searchable-collection-target="input" data-action="input->searchable-collection#input" placeholder=@text[:placeholder] aria-label=@text[:aria_label] aria-expanded="true" aria-describedby=@text[:aria_describedby] aria-owns="subjects__listbox" role="combobox"
+        input.govuk-input.icon.icon--left.icon--search.js-action data-searchable-collection-target="input" data-action="input->searchable-collection#input" placeholder=@text[:placeholder] aria-label=@text[:aria_label] aria-expanded="true" aria-describedby=@text[:aria_describedby] aria-controls="subjects__listbox" role="combobox"
```

Comment:
Apparently, this is neither valid as introduces a new issue:
![image](https://github.com/DFE-Digital/teaching-vacancies/assets/1227578/3536dc01-c4a0-4c09-97f5-57c7729cec97)

Reply:
Oh, this is fun. The issue is detected because the `subjects__listbox` id doesn't exist until a value is introduced on the subjects searchbox. Then the `id` gets added dynamically and the accessibility issue disappears:

![image](https://github.com/DFE-Digital/teaching-vacancies/assets/1227578/d85ac4f6-63e3-455e-ba89-28d5b38b7c9f)

 How can we avoid it? :thinking: